### PR TITLE
Add information on CSP [SC-314]

### PR DESCRIPTION
**Pull request by MarkvanMents**
@feddovanede 

Can you let me know if this covers the CSP documentation issue in [SC-314]?

### CODE REVIEW

File: content/developerportal/deploy/security-checklist-for-your-on-premises-installation.md [hunk 1]
```diff
@@ -56,11 +56,17 @@ The admin port will by default only allow connections from the local host. In ca
 
 Choose a strong password to protect the administrative interface running on the admin TCP port. Set this password to a long random string (when using the Windows Service Console, this is automatically done.) It is not used manually anywhere, and it is only used in the background by administrative tools like m2ee-tools and the Windows Service Console to be able to connect back to the Mendix application after it has been started for administrative tasks.
 
-## 9 Do Not Connect to a Production Database Using Studio Pro
+## 9 Setting the Content Security Policy Header
```
Comment (feddovanede) [inline content/developerportal/deploy/security-checklist-for-your-on-premises-installation.md]: Maybe the title should be more generic? _Setting Security headers_, for example? Content seems to cover the documentation issue!

Comment (MarkvanMents) [reply content/developerportal/deploy/security-checklist-for-your-on-premises-installation.md]: Yes - I wondered about that. Good suggestion.

**Comment by MarkvanMents** [comment]
@feddovanede - OK to merge, then?

**Comment by feddovanede** [comment]
@MarkvanMents Yes, looks great!

**Pull request merged**